content/en/seax/seax_messaging/seax-widget.md

```diff
@@ -0,0 +1,200 @@
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/product-updates/seax/zh/popout-input.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seax/zh/popout-input.png" alt="">
+    </a>
+    <p style="margin-top: 20px; font-size: 15px">Customize the style of your popout chat</p>
+</div>
+</div>
+
+Scroll further on the same page to set up your chat. Make sure to provide a opening message to greet your customers when they start a chat with you via SeaX.
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/product-updates/seax/zh/chat-setup.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seax/zh/chat-setup.png" alt="">
+    </a>
+    <p style="margin-top: 20px; font-size: 15px">Chat setup</p>
+</div>
+</div>
+
+That's it. Simply click on **Preview** window to see how your SeaX Widget will look like on your website or app. Once you are satisfied with the design, click on **Save** to apply the changes.
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/product-updates/seax/zh/popout-preview.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seax/zh/popout-preview.png" alt="">
+    </a>
+    <p style="margin-top: 20px; font-size: 15px">Preview your design</p>
+</div>
+</div>
+
+### Custom Forms
+Custom form is a powerful feature that allows you to collect information from your new customers before they start a chat with you via SeaX. You can create a form with different types of fields such as text, email, and phone number. SeaX will record these form responses as well the entire conversation history on SeaX. To create a custom form, click on **Add New Form** in the dashboard.
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/product-updates/seax/zh/form-setup.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seax/zh/form-setup.png" alt="">
+    </a>
+    <p style="margin-top: 20px; font-size: 15px">Customize the style of your custom forms</p>
+</div>
+</div>
+
+#### Set up Form
+
+1. **Enable This Form** - Turn on the switch to enable the form. Here you can also set the form's name and the title that the user will see.
```

Comment:
@itIsJim can you please add a screenshot of the form configuration page? It'd be easier for the users to follow along where these 3 options are on the UI